__tests__/Unit/Components/Searchbar/searchbar.test.tsx

```diff
@@ -0,0 +1,64 @@
+import Searchbar from '@/components/Dashboard/Searchbar';
+import { splitNSearch } from '@/utils/splitNSearch';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+jest.mock('../../../../src/utils/splitNSearch', () => ({
+    splitNSearch: jest.fn(),
+}));
+
+describe('test searchbar component', function () {
+    it('renders searchbar input and a search button', function () {
+        render(<Searchbar label="test-label" />);
+
+        const input = screen.getByTestId('searchbar_input') as HTMLInputElement;
+        const searchBtn = screen.getByTestId('search_btn') as HTMLButtonElement;
```

Comment:
According to the documentation, `ByTestId` API is recommended to be be used when the other queries are not useful to test.  Refer: https://testing-library.com/docs/queries/bytestid/#api
I believe in this case, you can use `ByRole` API instead. Kindly try out if you are able to use other queries first.

Reply:
I tried but won't getting things by role make the component brittle, example tomorrow I decide to add 1 more button

Reply:
Not necessarily,  you can customize your `ByRole` API to return specific button.
https://testing-library.com/docs/queries/byrole/#api

Reply:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text